projects/plugins/jetpack/tests/e2e/specs/search.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Internal dependencies
+ */
+import { syncJetpackPlanData } from '../lib/flows/jetpack-connect';
+import { activateModule, execWpCommand } from '../lib/utils-helper';
+import Homepage from '../lib/pages/homepage';
+import { step } from '../lib/env/test-setup';
+import {
+	enableInstantSearch,
+	setupSearchSidebarWidget,
+	setResultFormat,
+} from '../lib/search-helper';
+
+describe( 'Search', () => {
+	beforeAll( async () => {
+		await syncJetpackPlanData( 'complete' );
+		await activateModule( page, 'search' );
+		await enableInstantSearch();
+		await setupSearchSidebarWidget();
+	} );
+
+	afterAll( async () => {
+		await execWpCommand( 'wp jetpack module deactivate search' );
+	} );
+
+	it( 'Can perform search with default settings', async () => {
+		const homepage = await Homepage.visit( page );
+		await homepage.registerRouteInterceptions();
```

Comment:
Seems like all the tests calls these methods at the beginning. Is it possible to extract these into a `BeforeEach` hook?

Reply:
fixed in https://github.com/Automattic/jetpack/pull/19447/commits/5797898f9cc1835e204f8328b24b3469705dbc37